plutus-ledger/src/Ledger/Tx.hs

```diff
@@ -60,41 +59,53 @@ module Ledger.Tx
     , unspentOutputsTx
     -- * Hashing transactions
     , txId
+    -- * ChainIndexTxOut
+    , TxOutTx (..)
+    , txOutTxDatum
     ) where
 
 import Cardano.Api qualified as C
 import Cardano.Crypto.Hash (SHA256, digest)
 import Cardano.Crypto.Wallet qualified as Crypto
 import Codec.CBOR.Write qualified as Write
 import Codec.Serialise (Serialise (encode))
-import Control.Lens (At (at), makeLenses, makePrisms, (&), (?~))
+import Control.Lens (at, makeLenses, makePrisms, (&), (?~))
 import Data.Aeson (FromJSON, ToJSON)
 import Data.Map (Map)
 import Data.Map qualified as Map
-import Data.Maybe (mapMaybe)
+import Data.Maybe (isJust, mapMaybe)
 import Data.OpenApi qualified as OpenApi
 import Data.Proxy (Proxy (Proxy))
-import Data.Set (Set)
 import Data.Set qualified as Set
 import GHC.Generics (Generic)
 import Ledger.Address (Address, PaymentPubKey, StakePubKey, pubKeyAddress)
 import Ledger.Crypto (Passphrase, signTx, signTx', toPublicKey)
 import Ledger.Orphans ()
 import Ledger.Slot (SlotRange)
-import Ledger.Tx.CardanoAPI (SomeCardanoApiTx (SomeTx), ToCardanoError (..))
-import Ledger.Tx.CardanoAPI qualified as CardanoAPI
-import Ledger.Tx.Internal hiding (updateUtxoCollateral)
 import Ledger.Validation qualified
 import Plutus.Script.Utils.Scripts (datumHash)
 import Plutus.V1.Ledger.Api qualified as V1
 import Prettyprinter (Pretty (pretty), braces, colon, hang, nest, viaShow, vsep, (<+>))
 
 -- for re-export
-import Plutus.V1.Ledger.Tx hiding (TxIn (..), TxInType (..), inRef, inScripts, inType, pubKeyTxIn, pubKeyTxIns,
-                            scriptTxIn, scriptTxIns)
-import Plutus.V1.Ledger.Tx qualified as V1.Tx hiding (TxIn (..), TxInType (..))
+-- for re-export
```

Comment:
Something went wrong here. The line is duplicated and we're not exporting `Set`.

Reply:
Fixed by "Fix imports."